src/toil/test/server/wes_cwl_runner.py

```diff
@@ -0,0 +1,165 @@
+import argparse
+import json
+import logging
+import os
+import subprocess
+import sys
+import time
+from typing import Optional, Dict, Any, List
+
+from toil.test.server.wes_client import WESClient
+
+"""
+A CWL runner that submits a workflow to a WES server, waits for it to finish,
+and outputs the results.
+
+Example usage with cwltest:
+
+```
+cwltest --verbose \
+    --tool=python \
+    --test=cwl-v1.2-main/conformance_tests.yaml \
+    -n=0-197 \
+    --timeout=2400 \
+    -j1 \
+    -- \
+    src/toil/test/server/wes_cwl_runner.py \
+    --wes_endpoint=http://localhost:8080 \
+    --wes_user=test \
+    --wes_password=password \
+    --disableCaching \
+    --clean=always \
+    --logDebug
+```
+"""
+
+logger = logging.getLogger(__name__)
+
+
+def get_deps_from_cwltool(cwl_file, input_file, input_deps: bool = False):
+    """
+    Return a list of dependencies of the given workflow from cwltool.
```

Comment:
What does `input_deps` do here? Can we get only either the deps for the input file or the CWL file? Why do we need the input file if we aren't getting deps for it?

Reply:
> Can we get only either the deps for the input file or the CWL file?

I think so. We have to call `cwltool --print-deps cwl_file` to get the dependencies inside the CWL file, and call `cwltool --print-input-deps cwl_file input_file` to get the dependencies inside the input file.

I intended this to be two function calls based on `input_deps`, but I now realize that we don't need `input_deps` as we can just infer it if `input_file` is provided. Or, perhaps we could also execute them both here so we only need to call this function once.